graphql2/schema.graphql

```diff
@@ -525,6 +533,19 @@ input AlertSearchOptions {
   omit: [Int!]
 }
 
+input AlertLogSearchOptions {
+	first: Int = 15
+	after: String = ""
+	filterByAlertID: Int
+	filterByServiceID: String
+	filterByUserID: String
+	filterByIntegrationKeyID: String
```

Comment:
Should these take arrays instead of single values? Especially for `event`, I could see `[created, duplicate]` being a valid query.

Updating the SQL query to use `any( ... )` should make that work.

What do you think?